src/libPMacc/include/memory/boxes/SharedBox.hpp

```diff
@@ -22,13 +22,10 @@
 
 #pragma once
 
-#include "types.h"
-#include "math/Vector.hpp"
-
-
 #include <cuSTL/cursor/compile-time/BufferCursor.hpp>
 #include <math/vector/Float.hpp>
-
+#include "math/Vector.hpp"
```

Comment:
I think the include order is right but the question is why one include is written with <> and the other with ""?
Is there a correct guidline for this? I would prefer <> for everything because we never (should) use relative inclusion.